README.md

```diff
@@ -22,8 +22,22 @@ You need a repo to hold the log. Make this.
         */5 * * * * cd /opt/git/finglonger-tasks && flock -n /var/run/finglonger.lock git pull >> /var/log/finglonger.log 2>&1
         ```
 
+## finglonger-tasks repo
 
-# log format
+You need a finglonger tasks repo. Start out with:
+
+```
+envs/default/tasks.yaml
+files
+scripts
+```
+
+See https://github.com/nibalizer/finglonger-tasks as an example
+Tasks will only be run in the environment they are set in.
+
+
+
```

Comment:
Is this third new line on purpose? I don't see three new lines anywhere else in the file.